go.mod

```diff
@@ -1,12 +1,12 @@
 module github.com/ava-labs/subnet-evm
 
-go 1.23.6
+go 1.23.9
 
 require (
 	github.com/VictoriaMetrics/fastcache v1.12.1
 	github.com/antithesishq/antithesis-sdk-go v0.3.8
-	github.com/ava-labs/avalanchego v1.13.1-0.20250327151600-3a6bfac46f43
-	github.com/ava-labs/libevm v1.13.14-0.2.0.rc.4
+	github.com/ava-labs/avalanchego v1.13.1-rc.2.0.20250509202143-53d0f1d7fdb8
```

Comment:
Should we have a separate PR to update the avalanchego version? I think most of this PR is just small unrelated tweaks (except for the database package changes)